gpu-service/main.py

```
@@ -104,31 +111,39 @@ def lookup():
+                    # Ideally we select COSINE_SIMILARITY(...) as score, but let's keep it simple first.
+                    match = match_metadata
+                    
+                    try:
+                        image_obj = minio_client.get_object(match['bucket'], match['object_name'])
+                        image_bytes = image_obj.read()
+                        encoded_img = base64.b64encode(image_bytes).decode('utf-8')
+                        content_type, _ = mimetypes.guess_type(match['object_name'])
+                        if not content_type or not content_type.startswith('image/'):
+                            content_type = 'image/jpeg'
+                        match['image_data_uri'] = f"data:{content_type};base64,{encoded_img}"
+                    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)